### PR TITLE
Shalon

### PR DESCRIPTION
**Pull request by Kobsharon**

### CODE REVIEW

File: src/server/config/db.properties [hunk 1]
```diff
@@ -1,9 +1,14 @@
 # POSTGRES SQL parameters
 user=postgres
+<<<<<<< HEAD
 password=postgres
 url=jdbc:postgresql://localhost:5432/finalversion
 #
 #
+=======
+password=admin@123
+url=jdbc:postgresql://localhost:5432/chat_system_db
```
Comment (didiermunezero) [inline src/server/config/db.properties:10]: arent these erros

File: out/production/chat-system-java/server/config/db.properties [hunk 2]
```diff
@@ -1,9 +1,7 @@
 # POSTGRES SQL parameters
 user=postgres
-password=postgres
-url=jdbc:postgresql://localhost:5432/finalversion
-#
-#
+password=admin@123
```
Comment (didiermunezero) [inline out/production/chat-system-java/server/config/db.properties]: i think they are supposed to be

File: chat-system-java.iml [hunk 3]
```diff
@@ -3,14 +3,11 @@
   <component name="NewModuleRootManager" inherit-compiler-output="true">
     <exclude-output />
     <content url="file://$MODULE_DIR$">
-      <sourceFolder url="file://$MODULE_DIR$" isTestSource="false" />
       <sourceFolder url="file://$MODULE_DIR$/src" isTestSource="false" />
     </content>
     <orderEntry type="inheritedJdk" />
     <orderEntry type="sourceFolder" forTests="false" />
+    <orderEntry type="library" name="lib1" level="project" />
```
Comment (didiermunezero) [inline chat-system-java.iml:10]: cool

**Review by didiermunezero — COMMENTED** [review]
gonna check

File: src/server/config/db.properties (hunk 1 above)
Comment (Kobsharon) [reply src/server/config/db.properties:10]: already solved

**Pull request merged**